src/test/scala/fi/aalto/cs/apluscourses/intellij/actions/ReplActionTest.scala

```diff
@@ -6,9 +6,10 @@ import fi.aalto.cs.apluscourses.intellij.TestHelperScala
 import fi.aalto.cs.apluscourses.intellij.services.PluginSettings
 import fi.aalto.cs.apluscourses.intellij.utils.ModuleUtils
 import org.junit.Assert._
-import org.junit.Test
+import org.junit.{Ignore, Test}
 import org.mockito.Mockito.{mock, when}
 
+@Ignore
```

Comment:
Needs a `// temporary fix` comment that can be laughed at after a few years :smile:

Reply:
yepp, temporary)

Reply:
I'm sure that the remark of passed tests in each run of tests is enough to keep this issue in mind.